src/renderer/src/assets/main.css

```
@@ -3036,51 +3036,20 @@ html.onboarding-tour-start-transition::view-transition-new(root) {
-/* Why: a tour panel must read as elevated above the chrome it is teaching about
-   without resorting to color. A 1px inner top highlight plus the documented
-   floating shadow keeps it above the surface in both light and dark modes. */
-.orca-contextual-tour-panel {
-  background: linear-gradient(
-    180deg,
-    color-mix(in srgb, var(--foreground) 4%, var(--popover)) 0%,
-    var(--popover) 12%
-  );
+  --contextual-tour-panel-surface: color-mix(in srgb, var(--foreground) 7%, var(--popover));
+  --contextual-tour-panel-border: color-mix(in srgb, var(--foreground) 14%, var(--border));
+  background: var(--contextual-tour-panel-surface);
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)